platforms/windows/hax_entry.c

```diff
@@ -714,6 +715,16 @@ NTSTATUS HaxDeviceControl(PDEVICE_OBJECT DeviceObject,
             infret = sizeof(uint32_t);
             ret = STATUS_SUCCESS;
             break;
+        case HAX_IOCTL_CAP_MAX_VCPU:
+            if (outBufLength < sizeof(uint32_t)) {
+                Irp->IoStatus.Information = 0;
```

Comment:
Irp->IoStatus.Information is changed after done:
done:
    Irp->IoStatus.Status = ret;
    Irp->IoStatus.Information = infret;
infret is zero after init, so this line (Irp->IoStatus.Information = 0;) is not needed.

Reply:
Thanks for your suggestion! I have updated the code~